hatchways/team-honey-cruller
# Selecting a winner

### What this PR does (required):
- When a contest is created, the owner of the contest will get a notification and an email. We will also schedule a function to execute at the deadline date that will send the owner another notification and email informing them that their deadline has passed and to choose a winner. We also will update the database to set "active: false" for that contest.

- When a user chooses a winner, we will:
- Insert a document into our Winner collection
- Delete all submissions for the contest
- Delete the contest
- Delete all images from AWS except for the chosen winner
- Send a notification and email to the winning artist

- Also limited the ability to only select a winner once the contest is over
- Will not display any contests with "active: false"
- Contest owner can see their completed contests from their profile page

### Screenshots / Videos (required):
![image](https://user-images.githubusercontent.com/59981106/127546144-fd261460-9b38-4097-9f54-91d18e6e2f79.png)

![image](https://user-images.githubusercontent.com/59981106/127546204-ee1197d7-d6a9-4f0d-8d52-5440b1039480.png)

![image](https://user-images.githubusercontent.com/59981106/127546263-8cd8588d-2dcd-4be3-bf2b-47312cd3b8ad.png)


### Any information needed to test this feature (required):
- You will need to create a contest with a valid email address.
- You will need to submit at least one design to that contest from another user with a valid email address
- You will need to wait until the contest deadline has passed (or manually change the "active" status on the contest in your DB)
- Then, as the contest owner, you need to go to your profile page, to the completed section, select your contest and pick a winner

### Any issues with the current functionality (optional):
- Stripe is not integrated into this PR. Brian will create another PR to integrate stripe.
